### ISSUE TITLE
Support .NET Core

### ISSUE THREAD

**Issue by nbarbettini** [aaubry/YamlDotNet#180]
It'd be great to have .NET Core support for YamlDotNet now that RC2 is here.

**Comment by aaubry** [comment]
Unless I am mistaken, there is already Support for .Net core. The NuGet package contains the necessary targets, and I successfully compiled an app using a previous release. 
Please validate and if .Net core support does not work, provide more details of what you tried and which problems you encountered. Thanks

**Comment by nbarbettini** [comment]
Trying to add YamlDotNet 3.8.0 to a .NET Core RC2 library project results in this error:

```
    Package YamlDotNet 3.8.0 is not compatible with netstandard1.3 (.NETStandard,Version=v1.3). Package YamlDotNet 3.8.0 supports:
      - dotnet (.NETPlatform,Version=v5.0)
      - net35 (.NETFramework,Version=v3.5)
      - portable-net45+win8+wp8+wpa81 (.NETPortable,Version=v0.0,Profile=Profile259)
    One or more packages are incompatible with .NETStandard,Version=v1.3.
```

The project would need to target some `netstandardX.X` in order to be installed in .NET Core projects. See here: http://dotnet.github.io/docs/core-concepts/libraries/libraries-with-cli.html#how-to-target-net-core

**Comment by nbarbettini** [comment]
For now, it's possible to work around by adding `imports: [ "dotnet" ]` to my project.json 👍

**Comment by adamhathcock** [comment]
Made a PR for RTM.  Would like to get rid of the import statement.

**Comment by sparksterz** [comment]
@nbarbettini I'm now trying to do this with .net core 1.1 but I'm unable to simply add `imports: [ "dotnet" ]` anywhere and have it succeed...is there something I'm missing?

**Comment by aensidhe** [comment]
@aaubry I've made a [dirty port](https://github.com/aaubry/YamlDotNet/compare/master...e-vote:master) of library and tests to preview4 tooling of MSBuild tools. If you need it, feel free to use it.

- Perfomance and sample are not ported
- Unity and portable not checked
- System.Drawing tests excluded from netcore tests

We plan to use it until official release of netcore support.

**Comment by Nothing4You** [comment]
Since #244 is done I think this can be closed

**Comment by aaubry** [comment]
I am closing this as the development of this was completed on [another branch](https://github.com/aaubry/YamlDotNet/pull/255), which has been merged into master. I have released version [4.2.0](https://www.nuget.org/packages/YamlDotNet/4.2.0) which contains the new `netstandard1.3` assembly.
If you find anything that is not as it should, please open an issue.
